apps/site-projects/src/components/modules/Resources/TeamResources/TeamResources.js

```diff
@@ -0,0 +1,41 @@
+import PeopleDisplay from "./PeopleDisplay";
+import { TeamResourcesContainer } from "./StyledTeamResources";
+
+function getMembersAndLeaders(team) {
+    const allTeam = [];
+
+    team.leaders.forEach(element => {
+        allTeam.push(element)
+    });
+
+    team.members.forEach(element => {
+        allTeam.push(element)
+    });
+
+    return allTeam
+}
+
+function TeamResources({ team }) {
+    const allTeam = getMembersAndLeaders(team)
```

Comment:
if you're just merging two arrays you can just use the spread operator instead of calling the function : 
```const allTeam = [...team.leaders,...team.members];```

Reply:
This is totally true, I've commited your nice suggestion and i'll merge rn